tracing/src/field.rs

```diff
@@ -1,4 +1,143 @@
-//! Structured data associated with `Span`s and `Event`s.
+//! Span and `Event` key-value data.
```

Comment:
Not sure what the formatting intention here is, but based off the rest of the section, I think you want:
```suggestion
//! `Span` and `Event` key-value data.
```